### PR TITLE
Heedls 564 set active filter as default

### PR DESCRIPTION
**Pull request by wilkinsm**
### JIRA link
[https://softwiretech.atlassian.net/browse/HEEDLS-564](https://softwiretech.atlassian.net/browse/HEEDLS-564)

### Description

- Set the "Active Status" filter to on and "true" by default
- Add sort by Delegate ID to the sorting options.

### Screenshots
image

-----
### Developer checks
(Leave tasks unticked if they haven't been appropriate for your ticket.)

I have:
- [x] Run the formatter and made sure there are no IDE errors.
- [x] Written tests for the changes (accessibility tests, unit tests for controller, data services, services, view models, etc)
- [x] Manually tested my work with and without JavaScript. Full manual testing guidelines can be found here: https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/6703648740/Testing
- [ ] Updated/added documentation in Swiki and/or Readme. Links (if any) are below:
  - [doc_1_here](link_1_here)
- [ ] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken.
- [x] Scanned over my own MR to ensure everything is as expected.

### CODE REVIEW

File: DigitalLearningSolutions.Web/Helpers/GenericSortingHelper.cs [hunk 1]
```diff
@@ -163,6 +163,13 @@ public class CourseDelegatesSortByOption : Enumeration
             nameof(CourseDelegate.PassRate)
         );
 
+        public static readonly CourseDelegatesSortByOption CandidateNumber = new CourseDelegatesSortByOption(
+            7,
+            nameof(CandidateNumber),
+            "Delegate ID",
```
Comment (wilkinsm) [inline DigitalLearningSolutions.Web/Helpers/GenericSortingHelper.cs:169]: Confirmed by KW on 7th Feb, that it should say "Delegate ID" for this candidate number option.

**Review by DanBloxham-sw — APPROVED** [review]
I think this is pretty much perfect.

**Review by SteveJacksonSoft — CHANGES_REQUESTED** [review]
Yep, just a suggested name change.

File: DigitalLearningSolutions.Web/Scripts/searchSortFilterAndPaginate/sort.ts [hunk 2]
```diff
@@ -72,6 +72,8 @@ export function getSortValue(
       return parseInt(getElementText(searchableElement, 'faq-weighting'), 10);
     case 'FaqId':
       return parseInt(getElementText(searchableElement, 'faq-id'), 10);
+    case 'CandidateNumber':
+      return getElementText(searchableElement, 'candidate-number').toLocaleLowerCase();
```
[the remark this answers is not in the document]
Comment (wilkinsm) [reply DigitalLearningSolutions.Web/Scripts/searchSortFilterAndPaginate/sort.ts]: Done

**Pull request merged**